apps/system/test/unit/task_manager_test.js

```diff
@@ -545,59 +581,10 @@ suite('system/TaskManager >', function() {
       });
 
       test('initial state', function() {
-        assert.equal(taskManager.position, 0,
-                    'initial position should be 0');
         assert.ok(taskManager.currentCard,
                   'has a truthy currentCard property');
       });
 
-      function undefinedProps(value) {
-        for (var key in value) {
-          if (typeof value[key] === 'undefined') {
-            return true;
-          }
-        }
-        return false;
-      }
-
-      test('applyStyle is called by swiping', function(done) {
-        var card = taskManager.getCardAtIndex(0);
-        var element = card.element;
-        var applyStyleSpy = this.sinon.spy(card, 'applyStyle');
-
-        waitForEvent(element, 'touchend').then(function() {
-          var callCount = applyStyleSpy.callCount;
-          assert.isTrue(callCount > 0,
-                        'card.applyStyle was called at least once');
-          assert.isFalse(applyStyleSpy.calledWith(sinon.match(undefinedProps)),
-            'card.applyStyle was not called with undefined properties');
-
-        }, failOnReject).then(function() { done(); }, done);
-
-        // Simulate a drag up that doesn't remove the card
-        element.dispatchEvent(createTouchEvent('touchstart', element, 0, 500));
-        element.dispatchEvent(createTouchEvent('touchmove', element, 0, 250));
-        element.dispatchEvent(createTouchEvent('touchend', element, 0, 450));
-      });
-
-      test('cards should be hidden for better performance', function() {
-        var card = taskManager.getCardAtIndex(0);
-        assert.equal(card.element.style.visibility, '');
-
-        var farAway = taskManager.getCardAtIndex(3);
-        assert.equal(farAway.element.style.visibility, 'hidden');
-      });
-
-      test('and shown when needed', function() {
-        taskManager.position = 3;
-        taskManager.alignCurrentCard();
-        var card = taskManager.getCardAtIndex(0);
-        assert.equal(card.element.style.visibility, 'hidden');
-
-        var farAway = taskManager.getCardAtIndex(3);
-        assert.equal(farAway.element.style.visibility, '');
-      });
-
```

Comment:
would be cool to add a basic test asserting that the number of cards in the dom is correct + their `style.left` value